### PR TITLE
Add API to get config variables

### PR DESCRIPTION
**Pull request by KavinduZoysa**
## Purpose
$subject

### CODE REVIEW

File: flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/ConfigVariablesManager.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+import io.ballerina.projects.Document;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Generates functions based on a given keyword.
```
Comment (nipunayf) [inline flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/ConfigVariablesManager.java:44]: Need to match the description

File: flow-model-generator/modules/flow-model-generator-ls-extension/src/main/java/io/ballerina/flowmodelgenerator/extension/request/ConfigVariablesGetRequest.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package io.ballerina.flowmodelgenerator.extension.request;
```
[the remark this answers is not in the document]
Comment (KavinduZoysa) [reply flow-model-generator/modules/flow-model-generator-ls-extension/src/main/java/io/ballerina/flowmodelgenerator/extension/request/ConfigVariablesGetRequest.java:19]: Added

File: flow-model-generator/modules/flow-model-generator-ls-extension/src/test/java/io/ballerina/flowmodelgenerator/extension/ConfigVariablesTest.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                ConfigVariablesTestConfig.class);
+
+        ConfigVariablesGetRequest request =
+                new ConfigVariablesGetRequest(sourceDir.resolve(testConfig.configFile()).toAbsolutePath().toString());
+        JsonObject configVariables = getResponse(endpoint, request);
+
+        if (!configVariables.equals(testConfig.configVariables())) {
+            ConfigVariablesTestConfig updatedConfig = new ConfigVariablesTestConfig(testConfig.configFile(),
+                    configVariables);
+            updateConfig(configJsonPath, updatedConfig);
```
[the remark this answers is not in the document]
Comment (KavinduZoysa) [reply flow-model-generator/modules/flow-model-generator-ls-extension/src/test/java/io/ballerina/flowmodelgenerator/extension/ConfigVariablesTest.java]: Added

File: flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/ConfigVariablesManager.java (hunk 1 above)
Comment (KavinduZoysa) [reply flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/ConfigVariablesManager.java]: Added

**Review by nipunayf — APPROVED** [review]

**Pull request merged**